#! /bin/sh -e
## 20_move_spamd.dpatch by Duncan Findlay <duncf@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Make spamc's pod refer to spamd(8) not spamd(1)

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@
diff -urNad /home/duncf/debian/spamassassin-old2/spamassassin-2.99pre3.0.0svn20040523duncf1/spamc/spamc.pod spamassassin-2.99pre3.0.0svn20040523duncf1/spamc/spamc.pod
--- /home/duncf/debian/spamassassin-old2/spamassassin-2.99pre3.0.0svn20040523duncf1/spamc/spamc.pod	2004-05-16 23:40:54.000000000 -0400
+++ spamassassin-2.99pre3.0.0svn20040523duncf1/spamc/spamc.pod	2004-05-24 00:11:48.000000000 -0400
@@ -169,7 +169,7 @@
 
 =head1 SEE ALSO
 
-spamd(1)
+spamd(8)
 spamassassin(1)
 Mail::SpamAssassin(3)
 
